index.html

```diff
@@ -1,44 +1,225 @@
-                    </div>
-                    <div class="description">
-                        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-                    </div>
-                    <div class="connect">
-                        <h3>LET'S CONNECT</h3>
-                    </div>
-                    <ul>
-                        <li><a href=""><img class="twitter" src="images/header//twiter.png" alt="twiter_logo"></a></li>
-                        <li><a href=""><img class="linkedIn" src="images/header//linkedin.png" alt="linkedIn_logo"></a></li>
-                        <li><a href=""><img class="m" src="images/header//m.png" alt="m_logo"></a></li>
-                        <li><a href=""><img class="github" src="images/header//github.png" alt="github_logo"></a></li>
-                        <li><a href=""><img class="ok" src="images/header//ok.png" alt="ok_logo"></a></li>
+    <header class="tool_bar">
+        <nav class="the_nav">
+            <a href="#" class="my_logo"><img src="images/header/desktop/myLogo.png" alt="Wali's-logo"></a>
+            <a href="#" class="menu_for_mbl"><img src="images/header/Union.png" alt="the_union"></a>
+            <ul class="themenu">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#"> About</a></li>
+                <li><a href=""> Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="port_sec" id="folio_sec">
+            <div class="background_1">
+                <h1>I'm Praveen Glad to see you!</h1>
+                <p class="me_1">I’m a software developer! I can help you build a product , feature or website Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me. </p>
+                <p class="lets_cnt">LET'S CONNECT</p>
+                <ul class="social_icons">
+                    <li> <a href="#"><img class="social" src="images/header/twiter.png" alt="twitter" width="20" height="20"></a>
+                    </li>
+                    <li> <a href="#"><img class="social" src="images/header/linkedin.png" alt="linkedin" width="20"
+                                height="20"> </a> </li>
+                    <li> <a href="#"><img class="social" src="images/header/m.png" alt="medium" width="20" height="20"></a>
+                    </li>
+                    <li> <a href="#"><img class="social" src="images/header/github.png" alt="github" width="20" height="20"></a>
+                    </li>
+                    <li> <a href="#"><img class="social" src="images/header/ok.png" alt="angellist" width="20" height="20">
+                        </a> </li>
+                </ul>
+            </div>
```

Comment:
- [ ] Kindly follow the Figma design as much as possible. For the headline section the `background-color` should be white not gray and the `background-image` should be like the one on Figma. At last; kindly don't forget to add `border-radius` to bottom left side of the section 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/194708995-d0f5af78-b763-4f7c-9cfd-c2ecf6271d32.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/194709002-ca124e4c-b32c-4606-b3e6-f397fe7cf493.png)